crates/wit-parser/src/lib.rs

```diff
@@ -305,7 +309,7 @@ pub struct Function {
     pub results: Results,
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
I believe what the PartialEq and not Eq constraint was expressing here is that equality is only valid on items from the same "world", and if you compare across worlds equality isn't necessarily meaningful.